internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -245,13 +245,54 @@ func TestAccClusterAdvancedCluster_pausedToUnpaused(t *testing.T) {
 	})
 }
 
+func TestAccClusterAdvancedCluster_advancedConfig_oldMongoDBVersion(t *testing.T) {
+	acc.SkipIfTPFAdvancedCluster(t)
+	var (
+		projectID   = acc.ProjectIDExecution(t)
+		clusterName = acc.RandomClusterName()
+
+		processArgs20240530 = &admin20240530.ClusterDescriptionProcessArgs{
+			DefaultReadConcern:               conversion.StringPtr("available"),
+			DefaultWriteConcern:              conversion.StringPtr("1"),
+			FailIndexKeyTooLong:              conversion.Pointer(false),
+			JavascriptEnabled:                conversion.Pointer(true),
+			MinimumEnabledTlsProtocol:        conversion.StringPtr("TLS1_1"),
+			NoTableScan:                      conversion.Pointer(false),
+			OplogSizeMB:                      conversion.Pointer(1000),
+			SampleRefreshIntervalBIConnector: conversion.Pointer(310),
+			SampleSizeBIConnector:            conversion.Pointer(110),
+			TransactionLifetimeLimitSeconds:  conversion.Pointer[int64](300),
+		}
+		processArgs = &admin.ClusterDescriptionProcessArgs20240805{
+			ChangeStreamOptionsPreAndPostImagesExpireAfterSeconds: conversion.IntPtr(-1), // this will not be set in the TF configuration
+			DefaultMaxTimeMS: conversion.IntPtr(65),
+		}
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 acc.PreCheckBasicSleep(t, nil, projectID, clusterName),
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             acc.CheckDestroyCluster,
+		Steps: []resource.TestStep{
+			{
+				Config:      configAdvanced(projectID, clusterName, "7.0", processArgs20240530, processArgs),
```

Comment:
we could use 6.0 as the latest version that is not EoL right? https://www.mongodb.com/legal/support-policy/lifecycles